src/Servers/IIS/IIS/test/Common.FunctionalTests/AppOfflineTests.cs

```diff
@@ -77,7 +77,8 @@ public async Task AppOfflineDroppedWhileSiteFailedToStartInShim_AppOfflineServed
             DeletePublishOutput(deploymentResult);
         }
 
-        [ConditionalFact(Skip = "https://github.com/aspnet/IISIntegration/issues/933")]
+        [ConditionalFact]
+        [Flaky("https://github.com/aspnet/IISIntegration/issues/933")]
```

Comment:
Delete this test. There has been no action on fixing https://github.com/dotnet/core-setup/issues/4646 and it isn't our problem.